rotkehlchen/tests/unit/test_ethereum_airdrops.py

```diff
@@ -177,7 +177,7 @@ def mock_requests_get(url: str, timeout: int = 0):  # pylint: disable=unused-arg
             f'{AIRDROPS_REPO_BASE}/airdrops/grain_iou.csv':
                 f'address,tokens\n{TEST_ADDR2},16301717650649890035791\n',
             f'{AIRDROPS_REPO_BASE}/airdrops/shutter.csv':
-                f'address,tokens\n{TEST_ADDR2},394857.029384576349787465\n',
+                f'address,tokens\n\n{TEST_ADDR2},394857.029384576349787465\n',
```

Comment:
Is this extra line to act as a test for someting?

Reply:
yeah this is for the case when there is empty lines in the CSV, len(row) < 2